modules/custom/dkan_metastore/src/Service.php

```diff
@@ -37,16 +40,20 @@ class Service implements ContainerInjectionInterface {
   public static function create(ContainerInterface $container) {
     return new Service(
       $container->get('dkan_schema.schema_retriever'),
-      $container->get('dkan_metastore.sae_factory')
+      $container->get('dkan_metastore.sae_factory'),
+      $container->get('plugin.manager.dkan_common.data_modifier')
```

Comment:
Does this feel like it should be part of the constructor? I believe our class can work without the plugins. The plugins are optional. Maybe we should use setter injection instead of constructor injection?

Reply:
Thanks for the suggestions. I opened PR #292 to address both of the above.